polaris-react/src/components/Navigation/Navigation.scss

```diff
@@ -851,85 +924,6 @@ $nav-max-width: 360px;
   list-style: none;
 }
 
-.ListSecondary {
```

Comment:
I moved everything from within `.ListSecondary` into `.SecondaryNavigation` since they share the same purpose.